Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Detector.h

```diff
@@ -306,7 +308,7 @@ class Detector : public o2::Base::Detector
     GeometryHandler *mGeometryHandler;
     MisalignmentParameter *mMisalignmentParameter;
 
-    V1Layer **mGeometry;   //! Geometry
+    V3Layer **mGeometry;   //! Geometry
```

Comment:
So if I understand correctly, this can be left as it is for the time being, and then I'll produce an updated version of the code using the proper STL coding and submit a second pull request: am I right ?